src/components/footer.tsx

```
@@ -1,44 +1,49 @@
+    case 'instagram':
+      return <FaInstagramSquare />;
+    case 'linkedin':
+      return <FaLinkedin />;
+    case 'bluesky':
+      return <FaBluesky />;
+    default:
+      return null;
+  }
+}
+
+const Footer: React.FC<{ footer: Footer }> = ({ footer }) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Footer'. Consider to delete it or rename it.

'Footer' is defined here:


(lint/suspicious/noRedeclare)